tox.ini

```diff
@@ -41,6 +41,11 @@ deps =
     vulture
 commands = vulture tendrl
 
+[testenv:prospector]
```

Comment:
please make a new PR for this along with an issue. Cant take this in this PR

Reply:
Done